util/evaluate_tools.py

```diff
@@ -68,10 +67,10 @@ def calculate_report(wav_filenames, labels, decodings, losses):
     # Order the remaining items by their loss (lowest loss on top)
     samples.sort(key=lambda s: s.loss)
 
-    # Then order by descending WER/CER
+    # Then order by ascending WER/CER
```

Comment:
Why do you change that ?

Reply:
I think the sorting should be in a way that you see the worst WER and the worst loss, instead of the worst WER with the best lost. Did not make sense for me...

This could also be done by sorting loss in reversed order but then i need to reverse it a second time for printing

Reply:
The worst WER with the best loss is to identify systemic issues, where the acoustic model is confident, yet the result is completely off the mark. This can point to transcription errors and stuff like that. Please don't change it.